tests/common/mod.rs

```diff
@@ -80,10 +80,10 @@ pub fn keepldr_exec<'a>(bin: &str, input: impl Into<Option<&'a [u8]>>) -> Output
 
 /// Returns a handle to a child process through which output (stdout, stderr) can
 /// be accessed.
-pub fn keepldr_exec_crate<'a>(
+pub fn keepldr_exec_crate(
     crate_name: &str,
     bin: &str,
-    input: impl Into<Option<&'a [u8]>>,
+    input: impl Into<Option<Vec<u8>>>,
```

Comment:
Heh, I see you hit a problem where a spawned thread wanted a 'static lifetime so you had to do all this rigamarole to convert it into an owned value. I'll ask about accepting the `thread::scoped` RFC which would have prevented the need for this, but in the meantime this is fine.

Reply:
yes :+1: well spotted